packages/sdk/src/internal/validators.ts

```diff
@@ -35,6 +36,7 @@ const BridgeCoreSchema = object({
   amount: numberLike,
   threshold: optional(number()),
   sourceChains: optional(array(number())),
+  enableSwaps: optional(boolean()),
```

Comment:
Just move this to `BridgeCoreWithRecipientSchema` and we are good to go

Reply:
done, sorry it's not exactly clear to me how these types are put together :thinking: